iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
+kickstart_file: "host-os.ks"
+hostos_package_group: "virtualization-hypervisor"
+package_group_list:
```

Comment:
Do we need groups like desktop-debugging, dial-up, gnome-*, guest-desktop-agents, internet-*, multimedia, x11?